src/components/layout/RegisterNameForm/RegisterNameForm.tsx

```diff
@@ -258,7 +258,14 @@ function RegisterNameForm() {
 
         <div className="flex flex-column" style={{ gap: '75px' }}>
           <NameTokenSelector
-            selectedTokenCallback={(id) => handlePDNTId(id?.toString())}
+            selectedTokenCallback={(id) =>
+              id
+                ? handlePDNTId(id?.toString())
```

Comment:
It seems we're never passing a null value to this method. I'd suggest making the parameter of `handlePDNTId` to be non-nullable. Also, by this point id is non-nullable :)
```suggestion
                ? handlePDNTId(id.toString())
```

Reply:
fixed in https://github.com/ar-io/pdns-react/pull/229/commits/4de5520d092e8e98eaa9dcffe69150aac929361b